planning/motion_velocity_planner/autoware_motion_velocity_planner_node/src/planner_manager.hpp

```diff
@@ -48,7 +52,16 @@ class MotionVelocityPlannerManager
     const std::vector<autoware_planning_msgs::msg::TrajectoryPoint> & ego_trajectory_points,
     const std::shared_ptr<const PlannerData> planner_data);
 
+  // Diagnostic
+  std::shared_ptr<DiagnosticStatus> makeDiagnostic(
+    const std::string & reason, const bool is_decided = true);
+  void publishDiagnostics(
+    const rclcpp::Publisher<DiagnosticArray>::SharedPtr pub_ptr, const rclcpp::Time & current_time,
+    const bool publish_decided_diagnostics_only = true) const;
```

Comment:
what `publish_decided_diagnostics_only` is for?
this is always true because this function is called only
```
planner_manager_.publishDiagnostics(diagnostics_pub_, get_clock()->now(), true);
```

Reply:
And is this possible to change `publishDiagnostics` to  `getDiagnostics` ?
We may not need to give pub_ptr to manager, can put all publisher on the node side.

Reply:
Currently, this PR only publishes 'decision', so I'm setting it to true, but I'm not sure if there is other additional information that needs to be published at the moment, and if so, it may be necessary to set it to false always to publish diag instead of `publish_decided_diagnostics_only`.

Reply:
sure! let me change that.